api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2122,8 +2125,8 @@ public void copyObject(String bucketName, String objectName, String destBucketNa
    * @throws NoSuchAlgorithmException     upon requested algorithm was not found during signature calculation
    * @throws InvalidExpiresRangeException upon input expires is out of range
    */
-  public String presignedGetObject(String bucketName, String objectName, Integer expires,
```

Comment:
I think it would be better if we keep the old `presignedGetObject` method in place for backward compatibility reasons and have it call the newly introduced method `getPresignedObjectUrl` with `method='GET'` internally.